system-test/install.ts

```diff
@@ -30,9 +30,15 @@ describe('📦 pack-n-play test', () => {
         ts: readFileSync(
           './system-test/fixtures/sample/src/index.ts'
         ).toString(),
+        dependencies: ['typescript@4.8.3'],
       },
     };
-    await packNTest(options);
+    try {
+      await packNTest(options);
+    } catch (err) {
+      console.error('TS install failed:', err);
```

Comment:
I believe `err` here can be omitted, since it's already throwing in the next line, it should end up in `stderr` the same way:

```suggestion
      console.error('TS install failed');
```

Reply:
Before printing the `err`, the output of the errors being thrown was not including the output of the pack-n-play function, so was not possible to see where the error was happening. The output was just:

```
 Error: Process 227 exited with code 1.
      at ChildProcess.<anonymous> (node_modules/pack-n-play/build/src/utils.js:48:22)
          -> /workspace/node_modules/pack-n-play/src/utils.ts:62:11
      at maybeClose (internal/child_process.js:1022:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
 ```
 
Example: https://source.cloud.google.com/results/invocations/9334fa5b-5323-43b9-b948-d48731060481/targets/cloud-devrel%2Fclient-libraries%2Fnodejs%2Fpresubmit%2Fgoogleapis%2Fnodejs-bigquery-storage%2Fnode12%2Fsystem-test/log

Reply:
oh I see! it looks like `pack-n-play` is swallowing these errors then 🤔 

alright, in this case it sounds reasonable to me to have it printed here for sure! personally I'd put in the next line just to make it more readable, e.g: 

```suggestion
      console.error('TS install failed:\n', err);
```

but that's def an optional nit